src/_igraph/graphobject.c

```diff
@@ -3400,6 +3400,60 @@ PyObject *igraphmodule_Graph_Tree(PyTypeObject * type,
   return (PyObject *) self;
 }
 
+/** \ingroup python_interface_graph
+ * \brief Generates a random tree using one of a few methods.
+ *
+ * This method has three parameters:
+ * - n is the number of nodes in the tree.
+ * - directed is a bool that specifies if the edges should be directed. If so, they
+ * point away from the root.
+ * - method is one of:
+ *   - igraph.RANDOM_TREE_PRUFER (Prufer, directed currently unsupported)
+ *   - igraph.RANDOM_TREE_LERW (loop-erased random walk)
+ *
+ * \return a reference to the newly generated Python igraph object
+ * \sa igraph_tree
+ */
+PyObject *igraphmodule_Graph_Tree_Game(PyTypeObject * type,
+                                  PyObject * args, PyObject * kwds)
+{
+  long int n, directed_o, tree_method_o;
+  igraphmodule_GraphObject *self;
+  igraph_t g;
+
+  static char *kwlist[] = { "n", "directed", "method", NULL };
+
+  if (!PyArg_ParseTupleAndKeywords(args, kwds, "lll", kwlist,
+                                   &n, &directed_o, &tree_method_o))
```

Comment:
`directed` is a boolean so I'd rather go for `PyObject* directed_o` and use `PyObject_IsTrue(...)` for sake of consistency with how it's done elsewhere in the library.

As for the `method` argument, look into `convert.c` to see how it's typically done for other enums.

Reply:
alright let me take a look

Reply:
yep I've changed this now.